dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh

```diff
@@ -429,10 +471,6 @@ echo "[CHE] Deploying Che on ${OPENSHIFT_FLAVOR} (image ${CHE_IMAGE})"
 get_che_pod_config | oc apply --force=true -f -
 echo
 
-if [ "${CHE_DEDICATED_KEYCLOAK}" == "true" ]; then
-  "${COMMAND_DIR}"/multi-user/configure_and_start_keycloak.sh
```

Comment:
Is it possible not to run keycloak now?

Reply:
not sure I got the question, that section that I;ve removed was used to run KC after it failed first time now we don;t need that at all as it will start up normally with first attempt

Reply:
so yes it is possible not to run KC here because KC is already started at this moment, and even CHE deployed. that part was needed to trigger KC run after it failed for the first time and CHE is already deployed but now KC is already fully started before CHE deploy